index.html

```diff
@@ -2910,13 +2910,15 @@ <h3>Production</h3>
             </p>
 
             <p>
-The value of <code>@context</code> MUST be exactly one of these values.
+The value of <code>@context</code> MUST be registered in the DID
+Specification Registries [[?DID-SPEC-REGISTRIES]].
```

Comment:
This is a normative (and critical) dependency on the Registries, which is not possible as the Registries is not a normative specification.